autovalet_parking/src/autovalet_parking/Parker.py

```diff
@@ -223,29 +219,22 @@ def calculate_error(self):
         trans_err = (self.target_pose[:-1] - self.reached_pose[:-1])*100
         # Error in orientation (yaw) in degrees
         # make sure the orientation is the same in hardware so that the "270" works as well
-
-        align_tag_to_goal = 0
-        if self.park_direction == "left":
-            align_tag_to_goal = -90
-        elif self.park_direction == "right":
-            align_tag_to_goal = 90
+        align_tag_to_goal = -90 if self.park_direction == "left" else 90
         yaw_err   = np.abs((self.target_pose[2]*180/3.14 + align_tag_to_goal) - self.reached_pose[2]*180/3.14) #comparing the aruco tag (rotated by 270deg) with husky's yaw
-        if yaw_err>180:
-                yaw_err -=180
-        if yaw_err > 360:
+        print 'yaw err b4 wrapping', yaw_err
+        if yaw_err > 180:
+            yaw_err -= 180
+        if yaw_err > 300:
```

Comment:
why 300 and not 360? are you just assuming anything over 300 must be a wraparound thing?

Reply:
I got no idea Imma kill myself. @mishra-nikita I believe you wrote this piece of code? Do you remember why it's 300?

Reply:
*_*